### PR TITLE
updated Getting-Started.md - added more detail making the process easier to understand

### PR DESCRIPTION
**Pull request by sojohnnysaid**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -131,59 +131,91 @@ npm install --save fastify-plugin
...
+*/
+
 const fastifyPlugin = require('fastify-plugin')
-const MongoClient = require('mongodb').MongoClient
 
-async function dbConnector (fastify, options) {
-  const url = options.url
-  delete options.url
+async function dbConnector(fastify, options) {
+    const { ObjectId, MongoClient } = require('mongodb')
```
Comment (mcollina) [inline docs/Getting-Started.md:180]: I think we should use `fastify-mongodb` instead. I know this also teaches about our plugin system, but I think it's too much info for a "getting started".

Comment (sojohnnysaid) [reply docs/Getting-Started.md:180]: Ok i'll give it another try using fastify-mongodb. Thank you for taking a moment to look this over!

**Comment by sojohnnysaid** [comment]
@mcollina i've updated the examples to be more true to the original, added fastify-mongodb and a simpler call to the db.

**Comment by sojohnnysaid** [comment]
Thanks @mcollina! My first PR. 😅

File: docs/Getting-Started.md [hunk 2]
```diff
@@ -121,23 +121,20 @@ A typical solution is to use a complex callback, or promises - a system that wil
...
 ```
-npm install --save fastify-plugin
+npm i --save fastify-plugin fastify-mongodb
 ```
 
 **server.js**
 ```js
 const fastify = require('fastify')({
-  logger: true
+  logger: true,
```
Comment (jsumners) [inline docs/Getting-Started.md:134]:
```suggestion
  logger: true
```

File: docs/Getting-Started.md [hunk 3]
```diff
@@ -147,42 +144,48 @@ fastify.listen(3000, function (err, address) {
...
 
-async function dbConnector (fastify, options) {
-  const url = options.url
-  delete options.url
-
-  const db = await MongoClient.connect(url, options)
-  fastify.decorate('mongo', db)
+async function dbConnector(fastify, options) {
+  fastify.register(require('fastify-mongodb'), {
+    url: 'mongodb://localhost:27017/test_database',
```
Comment (jsumners) [inline docs/Getting-Started.md:156]:
```suggestion
    url: 'mongodb://localhost:27017/test_database'
```

File: docs/Getting-Started.md [hunk 4]
```diff
@@ -147,42 +144,48 @@ fastify.listen(3000, function (err, address) {
...
 const fastifyPlugin = require('fastify-plugin')
-const MongoClient = require('mongodb').MongoClient
 
-async function dbConnector (fastify, options) {
-  const url = options.url
-  delete options.url
-
-  const db = await MongoClient.connect(url, options)
-  fastify.decorate('mongo', db)
+async function dbConnector(fastify, options) {
```
Comment (jsumners) [inline docs/Getting-Started.md:154]:
```suggestion
async function dbConnector (fastify, options) {
```

File: docs/Getting-Started.md [hunk 5]
```diff
@@ -147,42 +144,48 @@ fastify.listen(3000, function (err, address) {
...
 module.exports = fastifyPlugin(dbConnector)
+
 ```
 
 **our-first-route.js**
 ```js
-async function routes (fastify, options) {
-  const database = fastify.mongo.db('db')
-  const collection = database.collection('test')
+async function routes(fastify, options) {
```
Comment (jsumners) [inline docs/Getting-Started.md:168]:
```suggestion
async function routes (fastify, options) {
```

**Comment by mcollina** [comment]
Good job and congrats for landing your first PR!

**Comment by sojohnnysaid** [comment]
Thanks and thank you for a great framework!

**Pull request merged**